README.md

```diff
@@ -193,22 +193,27 @@ The tool runs a series of analyzers to ensure submitted plugins are following be
 | ------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------- |
 | Archive Structure / `archive`                     | Ensures the contents of the zip file have the expected layout.                                                                    | None                                                                  |
 | Archive Name / `archivename`                      | The name of the archive should be correctly formatted.                                                                            | None                                                                  |
+| Backend Binary / `backendbinary` | Checks that binary exists for plugins declaring a backend in plugin.json | None |
 | Backend Debug / `backenddebug`                    | Checks that the standalone debug files for backend plugins are not present.                                                       | None                                                                  |
 | Binary Permissions / `binarypermissions`          | For datasources and apps with binaries, this ensures the plugin can run when extracted on a system.                               | None                                                                  |
 | Broken Links / `brokenlinks`                      | Detects if any URL doesn't resolve to a valid location.                                                                          | None                                                                  |
+| Checksum / `checksum` | Validates that the passed checksum (as a validator arg) is the one calculated from the archive file | `checksum` | 
```

Comment:
```suggestion
| Checksum / `checksum` | Validates that the passed checksum (as a validator arg) is the one calculated from the archive file. | `checksum` | 
```
Would you do this locally? I can't see where we describe how you would provide this when running outside of the plugin submission form